src/microbe_stage/PatchMap.cs

```diff
@@ -207,7 +207,10 @@ public void UpdateGlobalPopulations()
     /// <summary>
     ///   Removes species from patches where their population is &lt;= 0
     /// </summary>
-    public void RemoveExtinctSpecies(bool playerCantGoExtinct = false)
+    /// <returns>
+    ///   The extinct creatures
+    /// </returns>
+    public IEnumerable<Species> RemoveExtinctSpecies(bool playerCantGoExtinct = false)
```

Comment:
I think that it doesn't make much sense that this returns an enumerator, because I can see cases where the return value of this is ignored.
So I think that the return value should be `List<Species>` which would be built in the function body.